pkgs/development/python-modules/generic/default.nix

```diff
@@ -59,19 +49,18 @@ if disabled
 then throw "${name} not supported for interpreter ${python.executable}"
 else
 
+let
+  setuppy = ./run_setup.py;
```

Comment:
I'm still not sure what this is doing, but I've got to say I'm not a fan. This is going to confuse the heck out of developers trying to grok what `buildPythonPackage` is doing, and make error messages a lot more confusing. At the very least it needs some comments explaining the thinking.